src/patternfly/components/Login/docs/code.md

```diff
@@ -8,6 +8,7 @@
 | `.pf-c-login__header .pf-c-brand` | `<img>` |  Creates a brand image inside of login header. |
 | `.pf-c-login__main` | `<main>` |  Positions the login main area. |
 | `.pf-c-login__main-header` | `<header>` |  Creates the header of the main area. |
+| `.pf-c-login__main-header-desc` | `<p>` |  Creates the description in the main area header. |
```

Comment:
another question can you have `.pf-c-login__main-header-desc` without `.pf-c-login__main-header` and will that look ok? If not maybe add required to header?

Reply:
created https://github.com/patternfly/patternfly-next/issues/1572 to track that.